benchmarks/aggregate.py

```diff
@@ -94,13 +94,17 @@ def process_file(args, results_map: Dict[str, Any], filename: str):
         continue
       total_times = r['metrics']['total_time'] if 'total_time' in r[
           'metrics'] else []
-      if len(total_times) <= 1:
+      # Skip the first three elements in total_times[]; the first one
+      # includes compilation time, the 2nd and 3rd warm up the caches.
+      skip_elems = 3
+      if len(total_times) <= skip_elems:
         continue
-      # Skip total_times[0] because it includes compilation time.
-      avg = np.average(total_times[1:])
```

Comment:
George suggested using minimums, it's a worthy path to explore.

Reply:
Not sure I understand -- what does "using mininums" mean in this context?
Something we could do is to first discard outliers using statistics (e.g. anything too far beyond a certain amount of standard deviations). After that whether we picked the avg+std or something else should not make a big difference.